app-modules/portal/src/Http/Controllers/KnowledgeManagementPortal/KnowledgeManagementPortalController.php

```diff
@@ -75,6 +84,7 @@ public function show(): JsonResponse
                 )
             ),
             'footer_logo' => Vite::asset('resources/images/canyon-logo-light.png'),
+            'guest_id' => Session::get('guest_id'),
```

Comment:
What is the purpose of passing the `guest_id` here? If you need to retrieve it somewhere later in the code you should be able to reference it in the session right?

Reply:
Done